Makefile

```diff
@@ -44,6 +44,9 @@ run-prod:
 build-dev:
 	docker build --file=Dockerfile-dev . --tag openslides-backend-dev
 
+build-tests:
```

Comment:
Dockerfile for tests is placed in the tests directory.

Reply:
The next version has a different dockerfile integration.

Reply:
Please remove

Reply:
(ping)